packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -97,7 +105,11 @@ class _ScaffoldLayout extends MultiChildLayoutDelegate {
     }
 
     // The BottomSheet and the SnackBar are anchored to the bottom of the parent,
-    // they're as wide as the parent and are given their intrinsic height.
+    // they're as wide as the parent and are given their intrinsic height. The
+    // only difference is that SnackBar appears on the top side of the
+    // BottonNavigation while the BottomSheet occludes it by being layed out on
```

Comment:
layed => laid

You could just say that the BottomSheet is stacked on top of the  BottomNavigationBar.